assets/javascripts/kitten/components/form/location-input/index.js

```diff
@@ -215,4 +224,5 @@ LocationInput.propTypes = {
   inputProps: PropTypes.object,
   name: PropTypes.string,
   loadingText: PropTypes.string,
+  variant: PropTypes.string,
```

Comment:
Je mettrais carrément :

```suggestion
  variant: PropTypes.oneOf(['andromeda', 'orion']),
```